airbyte-webapp/src/scss/_variables.scss

```diff
@@ -37,3 +37,7 @@ $width-modal-sm: 492px;
 $width-modal-md: 585px;
 $width-modal-lg: 940px;
 $width-modal-xl: 1008px;
+
+$buttonHeightXs: 32px;
```

Comment:
Nit: the convention for var names in SCSS is `with-dashes`

Reply:
Thanks! Fixed that